cvat-ui/src/components/export-dataset/export-dataset-modal.tsx

```diff
@@ -168,10 +170,13 @@ function ExportDatasetModal(props: StateToProps): JSX.Element {
                 >
                     <Select virtual={false} placeholder='Select dataset format' className='cvat-modal-export-select'>
                         {dumpers
-                            .sort((a: any, b: any) => a.name.localeCompare(b.name))
-                            .filter((dumper: any): boolean => dumper.dimension === instance?.dimension)
+                            .sort((a: Dumper, b: Dumper) => a.name.localeCompare(b.name))
+                            .filter(
+                                (dumper: Dumper): boolean => dumper.dimension === instance?.dimension ||
+                                    instance instanceof Project,
```

Comment:
Hmm, does this condition work? 
For a project with 3D dimension it will show 2D formats because of the second part ``||  instance instanceof Project``

Reply:
Also optional chaining ``?.`` is not necessary here anymore, because all of Job, Task, Project have ``dimension`` property now.

Reply:
I see dimension is null for an empty project, so, we probably should check this value.
Like:
```
dumper.dimension === instance.dimension || (instance instanceof Project && instance.dimension === null)
```
What do you think?

<img width="824" alt="image" src="https://user-images.githubusercontent.com/40690378/226896808-f19fb5f0-771c-42a1-bd53-1e7bdc13e42a.png">

Reply:
Yep, checking for `null` is better.
BTW `istance` can be null, so ?. still needed. (Fixed instance type by adding `| null`)